### PR TITLE
Resolving - [ tud-669 ] -Fix on deleting some property from table to do not need to refresh page to apply new changes

### PR DESCRIPTION
**Pull request by Tsirov**

### CODE REVIEW

File: src/data/permissionsColumnData.js [hunk 1]
```diff
@@ -70,9 +71,19 @@ const permissionsColumnData = [
             <DialogActions>
               <Button
                 onClick={async () => {
-                  await deletePermission(deletePermissionId);
+                  searchParams.set('permission-management', deletePermissionId);
+
+                  await deletePermission(deletePermissionId)
+                    .then(() => {
```
Comment (S-Hristov) [inline src/data/permissionsColumnData.js:77]: Don't use .then use async awaut

File: src/data/rolesColumnData.js [hunk 2]
```diff
@@ -74,9 +75,19 @@ const rolesColumnData = [
             <DialogActions>
               <Button
                 onClick={async () => {
-                  await deleteRole(deleteRoleId);
+                  searchParams.set('role-management', deleteRoleId);
```
Comment (S-Hristov) [inline src/data/rolesColumnData.js:78]: Don't use .then use async awaut

File: src/layouts/player-management/index.js [hunk 3]
```diff
@@ -80,7 +84,7 @@ function PlayerManagement() {
     actions.setSubmitting(false);
     actions.resetForm();
   };
-
+  console.log('filtersssssssssssss', filters);
```
Comment (S-Hristov) [inline src/layouts/player-management/index.js:87]: remove logs

File: src/layouts/user-management/index.js [hunk 4]
```diff
@@ -49,9 +49,18 @@ function UserManagement() {
           <DialogActions>
             <MDButton
               onClick={async () => {
-                await deleteUser(deleteUserId);
+                await deleteUser(deleteUserId)
+                  .then(() => {
```
Comment (S-Hristov) [inline src/layouts/user-management/index.js:53]: Don't use .then use async awaut

File: src/services/users/index.js [hunk 5]
```diff
@@ -96,18 +96,7 @@ export const deleteUser = async (id) => {
...
-        firstName: '',
-        lastName: '',
-        email: '',
-        role: {
-          id: '',
-          name: ''
-        }
-      }
-    };
+    throw new Error(err.message);
```
Comment (S-Hristov) [inline src/services/users/index.js:99]: Either catch it and handle it or do not us try catch

File: src/services/roles/index.js [hunk 6]
```diff
@@ -104,12 +104,6 @@ export const deleteRole = async (id) => {
...
     return await api.delete(`/admin/auth/roles/${id}`, {});
   } catch (err) {
-    console.log(err);
-    return {
-      data: {
-        name: '',
-        permissions: []
-      }
-    };
+    throw new Error(err.message);
```
Comment (S-Hristov) [inline src/services/roles/index.js:107]: Either catch it and handle it or do not us try catch

File: src/services/players/index.js [hunk 7]
```diff
@@ -137,7 +137,7 @@ export const deletePlayer = async (id) => {
     });
   } catch (err) {
     console.error(`Error deleting user:`, err);
-    alert(`Error deleting user: ${err.message}`);
+    throw new Error(err.message);
```
Comment (S-Hristov) [inline src/services/players/index.js:140]: Either catch it and handle it or do not us try catch

File: src/services/permissions/index.js [hunk 8]
```diff
@@ -98,12 +98,6 @@ export const deletePermission = async (id) => {
...
     return await api.delete(`/admin/auth/permissions/${id}`);
   } catch (err) {
-    console.log(err);
-    return {
-      data: {
-        action: '',
-        object: ''
-      }
-    };
+    throw new Error(err.message);
```
Comment (S-Hristov) [inline src/services/permissions/index.js:101]: Either catch it and handle it or do not us try catch

**Review by S-Hristov — CHANGES_REQUESTED** [review]

File: src/data/permissionsColumnData.js (hunk 1 above)
Comment (Tsirov) [reply src/data/permissionsColumnData.js]: Fixed.

File: src/data/rolesColumnData.js (hunk 2 above)
Comment (Tsirov) [reply src/data/rolesColumnData.js:78]: Fixed.

File: src/layouts/player-management/index.js (hunk 3 above)
Comment (Tsirov) [reply src/layouts/player-management/index.js]: Fixed.

File: src/layouts/user-management/index.js (hunk 4 above)
Comment (Tsirov) [reply src/layouts/user-management/index.js]: Fixed.

File: src/services/users/index.js (hunk 5 above)
Comment (Tsirov) [reply src/services/users/index.js]: Fixed.

File: src/services/roles/index.js (hunk 6 above)
Comment (Tsirov) [reply src/services/roles/index.js]: Fixed.

File: src/services/players/index.js (hunk 7 above)
Comment (Tsirov) [reply src/services/players/index.js]: Fixed.

File: src/services/permissions/index.js (hunk 8 above)
Comment (Tsirov) [reply src/services/permissions/index.js]: Fixed.

**Pull request merged**